### DISCUSSION TITLE
【问题请教】是否支持静态资源文件包自动更新？

### DISCUSSION THREAD

**Discussion by mopduan** [tw93/Pake#669] [Q&A]
看到Pake支持打包静态资源文件包：[https://github.com/tw93/Pake/wiki/Pake-packaging-static-file-example](https://github.com/tw93/Pake/wiki/Pake-packaging-static-file-example)，Tauri支持Updater：[https://tauri.app/v1/guides/distribution/updater/](https://tauri.app/v1/guides/distribution/updater/)，但是未找到Tauri或Pake是否能支持自动更新静态资源文件包的文档说明，想了解下目前是否只能以整体安装包更新的方式来更新静态资源文件包？如果目前不支持的话，后续是否有考虑支持？谢谢！

**Comment by tw93** [comment]
假如静态文件是cdn，那就不需要资源包了，资源包比较累赘，不方便使用
